Makefile

```diff
@@ -10,11 +10,11 @@ test: test_no_proxy test_proxy
 
 test_no_proxy:
 	@echo "Running all tests with no proxy process"
-	CODEJAIL_PROXY=0 nosetests --with-xunit --xunit-file reports/nosetests-no-proxy.xml
+	CODEJAIL_PROXY=0 pytest --junitxml=reports/pytest-no-proxy.xml --log-level=DEBUG
```

Comment:
why we need to add this param `--log-level=DEBUG`?

Reply:
There was a test (https://github.com/edly-io/codejail/blob/master/codejail/tests/test_jail_code.py#L260) that was failing on pytest but passing on the nose framework. It was doing some logger assertions (log._log). My guess was that due to the difference in logging levels in the nose and pytest, some log messages were ignored. Upon changing the debug level, the test started passing.